addons/docs/src/mdx/mdx-compiler-plugin.js

```diff
@@ -162,19 +168,18 @@ function getExports(node, counter) {
 // insert `mdxKind` into the context so that we can know what "kind" we're rendering into
 // when we render <Story name="xxx">...</Story>, since this MDX can be attached to any `selectedKind`!
 const wrapperJs = `
-const mdxKind = componentMeta.title;
 componentMeta.parameters = componentMeta.parameters || {};
 componentMeta.parameters.docs = {
-  container: ({ context, children }) => <DocsContainer context={{...context, mdxKind}}>{children}</DocsContainer>,
+  container: ({ context, children }) => <DocsContainer context={{...context, mdxStoryNameToId}}>{children}</DocsContainer>,
   page: MDXContent,
 };
 `.trim();
 
-function stringifyMeta(meta) {
+function stringifyObject(obj) {
   let result = '{ ';
```

Comment:
Why not use json stringify

Reply:
```
    - const componentMeta = { title: 'docs-only', includeStories: ['storybookDocsOnly'] };
    + const componentMeta = { title: "'docs-only'", includeStories: '["storybookDocsOnly"]' };
```

Reply:
Updated to use `JSON.stringify` for `mdxStoryNameToId`. Can't use easily for `Meta`, so reverted that part of the change.